<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="D-184003189" name="Test2" tests="2" failures="2" timestamp="19 Feb 2018 07:39:17 GMT" time="221.404" errors="0">
  <testcase name="@AfterClass Final" time="0.0" classname="Selpack.Testop2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Selpack.Testop2.Final(Testop2.java:247)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass Final -->
  <testcase name="logincred" time="0.004" classname="Selpack.Testop2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Selpack.Testop2.logincred(Testop2.java:73)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- logincred -->
  <testcase name="searchtest" time="0.004" classname="Selpack.Testop2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Selpack.Testop2.searchtest(Testop2.java:91)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- searchtest -->
</testsuite> <!-- Test2 -->
